README.md

```diff
@@ -124,6 +124,24 @@ proc.aborted; // true
 proc.killed; // true
 ```
 
+### Using with args-tokenizer
```

Comment:
i would rename this to:

```suggestion
### Using with command strings
```

or something along the lines of that. it isn't specific to the library we use, rather its a way to allow people to continue using strings like `"cmd arg0 arg1 arg2"`

Reply:
you could then reword the below slightly to something like this:

> If you need to continue supporting commands as strings (e.g. `"command arg0 arg1"`), you can use args-tokenizer, a lightweight library for parsing shell command strings into an array.

Reply:
Sounds good